mantidproject/mantid
# SNSPowderReduction upgrade

Rework `SNSPowderReduction` and `PDtoPDFgetN` to take advantage of file caching.

**To test:**

Try out `SNSPowderReduction` in normal usage and see that things still work as expected.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
